src/Controllers/MoviesController.cs

```diff
@@ -171,5 +172,22 @@ public IActionResult SendEmail(MovieViewModel mvm)
             client.Send(mailMessage);
             return RedirectToAction("Index", "Movies");
         }
+
+        [HttpGet("Comment")]
+        public IActionResult Comment(int id)
+        {
+            var movie = UnitOfWork.Movies.Get(id);
+            var movieVM = new MovieViewModel { Id = movie.Id, Title = movie.Title, ReleaseDate = movie.ReleaseDate, Genre = movie.Genre, Price = movie.Price, Rating = movie.Rating };
+            return View(new CommentViewModel { Movie = movieVM });
+        }
```

Comment:
again, its ok, but you can generate a constructor to the MovieViewModel class and then:
`var movieVM = new MovieViewModel(UnitOfWork.Movies.Get(id));`